specs/__snapshots__/headingSpec.js.snap

```diff
@@ -42,7 +42,7 @@ exports[`Headings Should render heading 4 with default styles 1`] = `
 
 exports[`Headings Should render heading 5 with custom styles 1`] = `
 "%cHeading 5
- font-family:Times New Roman;font-size:13px;font-weight:bold;color:red;margin:12px 0px;"
+ font-family:Times New Roman;font-size:13px;font-weight:bold;color:black;margin:12px 0px;"
```

Comment:
Im just wondering why these tests changed. Has to be with something of the `children` check you do?

Reply:
Found the cause. Please read following comment